jetstream/experimenter.py

```diff
@@ -18,8 +18,60 @@ def _coerce_none_to_zero(x: Optional[int]) -> int:
     return 0 if x is None else x
 
 
+def _unix_millis_to_datetime(num: Optional[float]) -> Optional[dt.datetime]:
+    if num is None:
+        return None
+    return dt.datetime.fromtimestamp(num / 1e3, pytz.utc)
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class FeatureScalarTelemetry:
+    kind: str = "scalar"
+    name: str
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class FeatureEventTelemetry:
+    kind: str = "event"
+    event_category: str
+    event_method: Optional[str]
+    event_object: Optional[str]
+    event_value: Optional[str]
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class Feature:
+    slug: str
+    telemetry: Union[FeatureEventTelemetry, FeatureScalarTelemetry]
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class Branch:
+    slug: str
+    ratio: int
+
+
 @attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
 class Experiment:
+    """Common Experimenter experiment representation."""
+
+    slug: str
+    normandy_slug: Optional[str]
+    type: str
+    status: Optional[str]
+    active: bool
+    features: List[Feature]
+    branches: List[Branch]
+    start_date: Optional[dt.datetime]
+    end_date: Optional[dt.datetime]
+    proposed_enrollment: Optional[int]
+    reference_branch: Optional[str]
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class LegacyExperiment:
```

Comment:
I'm open to renaming this, e.g. we could also call this `ExperimentV1` and the Nimbus ones `ExperimentV4`